addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,55 @@
+# iCloud Binding
+
+The iCloud Binding helps to integrate apple device information into openHAB.
+
+## Supported Things
+
+The following devices are known to work with this binding:
+* iPhone 6s
+* iPhone 5c
+* iPhone 7
+* iPad Air 2/2017/Pro
+
+But other devices should work as well. Please provide feedback if you have tested another device type. We will add it to the list.
+
+## Discovery
+
+For each iCloud account to be included in openHAB a binding (bridge) needs to be configured with your apple id, password & refresh rate. 
+
+The devices registered to this account will then be automatically discovered.
+
+
+## Binding Configuration
+
+No textual configuration required/supported.
+
+## Thing Configuration
+
+Devices are automatically discovered once the binding (bridge) is configured.
+
+## Channels
+
+The following channels are available (if supported by the device):
+* Battery Status
+* Battery Level
+* Find My Phone
+* Location
+* Location Accuracy
+* Last Update
+* Address Street
+* Address City
+* Address Country
+* Formatted Address
```

Comment:
Turning this list into a table with more details might be helpful. See http://docs.openhab.org/addons/bindings/airquality/readme.html

Reply:
What is the right markup syntax for a table - tried several from the internet; but eclipse does not render it in the preview. Thus I´m not sure which one to use.